spec/controllers/tasks_controller_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+describe TasksController, type: :controller do
+  let(:user) { create(:user) }
+  let(:not_current_user) { create(:user) }
+  let(:task) { create(:task, user_id: user.id) }
```

Comment:
let's use `instance_double` instead of creating an actual entity, it's a quicker and more controlled way